### PR TITLE
Update action groups

### PR DESCRIPTION
**Pull request by s-hertel**
The format expected by https://github.com/ansible/ansible/pull/67291 changed, so just updating these here.

### CODE REVIEW

**Comment by openstack-gerrit** [comment]
Thank you for contributing to openstack/ansible-collections-openstack!

openstack/ansible-collections-openstack uses Gerrit for code review.

If you have never contributed to OpenStack before make sure you have read the
getting started documentation:
https://docs.openstack.org/infra/manual/developers.html#getting-started

Otherwise please visit
https://docs.openstack.org/infra/manual/developers.html#development-workflow
and follow the instructions there to upload your change to Gerrit.

File: meta/runtime.yml [hunk 1]
```diff
@@ -1,3 +1,278 @@
...
+  - os_user_info
+  - os_user_role
+  - os_user_role
+  - os_volume
+  - os_volume_snapshot
+  - os_zone
+  - os_zone
+  
+action_groups_redirection:
+  os:
```
Comment (sshnaidm) [inline meta/runtime.yml:273]: @s-hertel I have a question about redirect:
should it be instead:
```yaml
action_groups_redirection:
  openstack:
    redirect: os
```
If so, can we just drop here section of `openstack` and leave only `os`? It's actually almost duplicate each of other. (while `os` includes `openstack`)
Or maybe I miss something in redirection?
Thanks

Comment (s-hertel) [reply meta/runtime.yml:273]: The change was in flux still in ansible/ansible at the time, and `action_groups_redirection` needs to be removed from this now.

Comment (sshnaidm) [reply meta/runtime.yml:273]: Created patch in Gerrit:
Move action_group to runtime  https://review.opendev.org/735977

**Pull request closed without merging**